docs/devguide/contributing.rst

```diff
@@ -35,3 +35,76 @@ negative votes, the team will grant commit privileges.
 The core team will also be responsible for removing commit privileges when
 appropriate - for example for malicious merge behavior or just inactivity over
 an extended period of time.
+
+
+Quality gates for the pull requests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are three quality gates to ensure the pull requests quality, `Hound`_ for
```

Comment:
@yyscamper I wouldn't consider the style-checks or coveralls build gates.  These are really just hints to reviewers.  We frequently merge code with style or coverage warnings, but we should never be merging code that doesn't pass functional or unit tests.